Content.Server/Damage/PainSound/PainSoundSystem.cs

```diff
@@ -0,0 +1,94 @@
+using Content.Shared.Bed.Sleep;
+using Content.Shared.Damage;
+using Content.Shared.Damage.PainSound;
+using Content.Shared.FixedPoint;
+using Content.Shared.Humanoid;
+using Content.Shared.Mobs;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Stunnable;
+using Robust.Shared.Audio;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Timing;
+
+namespace Content.Server.Damage.PainSound;
+
+/// <inheritdoc cref="PainSoundComponent"/>
+public sealed class PainSoundSystem : EntitySystem
+{
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly IPrototypeManager _protMan = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+    private AudioParams _params = AudioParams.Default.WithMaxDistance(7).WithVolume(5);
+    private SoundCollectionPrototype? _painSoundMale;
+    private SoundCollectionPrototype? _painSoundFemale;
+    private SoundCollectionPrototype? _painSoundOther;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        _painSoundMale = _protMan.Index<SoundCollectionPrototype>("PainSoundsMale");
+        _painSoundFemale = _protMan.Index<SoundCollectionPrototype>("PainSoundsFemale");
+        _painSoundOther = _protMan.Index<SoundCollectionPrototype>("PainSoundsOther");
```

Comment:
Move this to your PainSound component. This would let us add sound collections on a species basis.